module/VuFind/src/VuFind/Config/ConfigManager.php

```diff
@@ -53,6 +53,8 @@ class ConfigManager
 {
     use MergeRecursiveTrait;
 
+    protected array $cache = [];
```

Comment:
This needs a comment, maybe like:

```suggestion
    /**
     * Configuration cache
     *
     * @var array[]
     */
    protected array $cache = [];
```